opentelemetry-sdk/tests/metrics/test_import.py

```diff
@@ -72,7 +72,7 @@ def test_import_view(self):
         """
 
         try:
-            from opentelemetry.sdk._metrics.view import (  # noqa: F401
+            from opentelemetry.sdk.metrics.view import (  # noqa: F401
```

Comment:
Feel free to address in a followup, but I don't think the `noqa` is of use here anymore